client/state/ui/test/reducer.js

```diff
@@ -91,4 +100,30 @@ describe( 'reducer', () => {
 			expect( state ).to.be.true;
 		} );
 	} );
+
+	describe( '#getSidebarIsCollapsed()', () => {
+		test( 'should default to false', () => {
+			const state = sidebarIsCollapsed( undefined, {} );
+
+			expect( state ).to.be.false;
+		} );
+
+		test( 'should be true when collapsed', () => {
+			const state = sidebarIsCollapsed( null, {
+				type: SIDEBAR_TOGGLE_VISIBILITY,
+				collapsed: true,
+			} );
+
+			expect( state ).to.be.true;
```

Comment:
If we have some initial state here where `collapsed` is definitely set to `false` then we can be more confident the action passing through the reducer actually causes it to toggle to `true`.

Reply:
Done https://github.com/Automattic/wp-calypso/pull/46045/commits/86a3dc8c4f5dc220f516fbaaf897d04294641592